recipes/unitpackage/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "unitpackage" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/unitpackage-{{ version }}.tar.gz
+  sha256: 5af8a74fc16558abf452e7d02cebd673aacd5a6a90ffb58d60752d700541dac4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - astropy >=5,<6
+    - filelock >=3,<4
+    - frictionless >=5.10.1,<6
+    - matplotlib-base >=3.5.0,<4
+    - pandas >=1,<2
+    - plotly >=5,<6
+    - pybtex >=0.24,<0.25
+    - svgdigitizer >=0.10.0,<0.11.0
+
+test:
+  imports:
+    - unitpackage
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/unitpackage/
+  summary: a Python library to work with the echemdb repository
```

Comment:
```suggestion
  summary: a Python library to interact with a collection of frictionless datapackages
```

Reply:
great thx, I updated this also in the repository